### DISCUSSION TITLE
Rechteck aufziehen - Auswahl - Markierung nicht sichtbar

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#59] [Q&A]
Kartenbild herunterladen mit Rechteck aufziehen:
Wenn man einen Bereich mit Rechteck aufziehen auswählt, verschwindet die Markierung. Man weiß nicht mehr, welchen Bereich man gewählt hat. 
Tritt überall auf, wo Rechteck aufziehen als Auswahlwerkzeug wählbar ist.
Schön wäre es, bliebe die Markierung erhalten.

![rechteckauswahl](https://user-images.githubusercontent.com/100088983/233946670-153523af-908c-4d6d-884d-a6e5833481f4.png)

**Comment by gis-eni** [comment]
Bin mir nicht sicher, ob das überall gewünscht wird. Nach einer Auswahl werden beispielsweise eh die betroffene Objekte selektiert. Das wäre eher störend. Hatten wir noch nie und hab ich auch sonst noch nie gesehen.
